src/lib/hooks/useLedgerStatus.ts

```diff
@@ -1,5 +1,5 @@
 export function useLedgerStatus() {
-  if (!window) return { isRunningInLedgerLive: false }
+  if (typeof window === 'undefined') return { isRunningInLedgerLive: false }
```

Comment:
Oh 👀 this fixes that annoying error/warning? nice!

Reply:
@janndriessen maybe?! I saw it surface via a build failure: https://vercel.com/index-coop/index-app/9uhEv1buM5Dzob62mP7vihoYhGEB#L88

Is that where you saw the error/how did it surface for you?

Reply:
I sometimes saw this in the console. Not 100 percent sure it's the same but good fix nevertheless.